inputfiles/idl/WebGL WEBGL_compressed_texture_atc.widl

```diff
@@ -0,0 +1,7 @@
+[NoInterfaceObject]
+interface WEBGL_compressed_texture_atc {
+    /* Compressed Texture Format */
+    const GLenum COMPRESSED_RGB_ATC_WEBGL = 0x8C92;
+    const GLenum COMPRESSED_RGBA_ATC_EXPLICIT_ALPHA_WEBGL = 0x8C92; // TODO: Is this a bug on the MDN site?
```

Comment:
This is rejected by https://github.com/KhronosGroup/WebGL/pull/2641, probably why it's wrong.